src/tools/sdk/_windows.md

```diff
@@ -1,23 +1,41 @@
+###How to install?
+
 You can install the Dart SDK using [Chocolatey.][Chocolatey]
 
-{{site.alert.important}}
-These commands require administrator privileges.
-If you need help on starting an administrator-level command prompt,
-try a search like
-<em><a href="https://www.google.com/search?q=cmd+admin"
-  target="blank">cmd admin</a>.</em>
-{{site.alert.end}}
+> **_NOTE:_** These commands require administrative privileges. To start command prompt as an administrator following these steps:
+> 
+> - Press Windows+R to open the “Run” box. 
+> - Type “cmd” into the box 
+> - Press Ctrl+Shift+Enter to run the command as an administrator.
 
-To install the Dart SDK:
+####To install the Dart SDK:
 
 ```terminal
 C:\> choco install dart-sdk
 ```
 
-To **upgrade** the Dart SDK:
+#####To upgrade the Dart SDK:
 
 ```terminal
 C:\> choco upgrade dart-sdk
 ```
 
-[Chocolatey]: https://chocolatey.org
+####Default installation directory:
+
+```terminal
+C:\tools\dart-sdk
+```
+
+###How to change installation directory?
+
+This package uses the help Get-BinRoot, which means that you can set an environment variable called ChocolateyToolsLocation, which controls where this package is extracted to. For more information click <a ahref=https://docs.chocolatey.org/en-us/create/functions/get-toolslocation>here</a>.
```

Comment:
For this section and other parts of the document, we try to stick to under 80 characters long per line and use semantic line breaks to accomplish this. You can read more about how to do this here: https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks

Let me know if you have any questions about doing so.

Also for links, use the [Markdown link syntax](https://www.markdownguide.org/basic-syntax/#links) and try to create a more thoughtful statement instead of "click here". You can learn more about this in the documentation for [References to other documents](https://developers.google.com/style/cross-references?hl=en#references-to-other-documents).

Reply:
Also I'm not super sure I understand the "help Get-BinRoot" piece of this instruction. What are you trying to convey there?

Reply:
Ok i'll do that.

And as for "Get-BinRoot", it's an inbuilt environment variable which Chocolatey uses for some packages which allows the user to change the installation directory. 

So our package dart-sdk uses that as I checked it on its page on Chocolatey. So that was just a minor detail about the package for people familiar with Chocolatey installations so that they find it easier to change the installation directory.

Reply:
Thanks for the context. I'm not sure all the extra details here are necessary for a user to properly install Dart and may actually cause confusion. So is the essential idea that users can configure the `ChocolateyToolsLocation` environment variable to change the install location? If so, perhaps we can simply state something about that, and after that direct users with a statement like "For more information, see [Chocolatey's GET-TOOLSLOCATION documentation](https://docs.chocolatey.org/en-us/create/functions/get-toolslocation).".

Reply:
sure